packages/mask/src/plugins/Wallet/SNSAdaptor/SelectProviderDialog/PluginProviderRender.tsx

```diff
@@ -218,3 +230,38 @@ export function PluginProviderRender({
         </>
     )
 }
+
+interface NetworkItemProps {
+    onNetworkIconClicked: PluginProviderRenderProps['onNetworkIconClicked']
+    NetworkIconClickBait?: PluginProviderRenderProps['NetworkIconClickBait']
+    network: Web3Helper.NetworkDescriptorAll
+    undeterminedNetworkID: string | undefined
+}
+
+function NetworkItem({ onNetworkIconClicked, NetworkIconClickBait, network, undeterminedNetworkID }: NetworkItemProps) {
```

Comment:
So where is the `<ShadowRootTooltip />`?

Reply:
Tooltip is removed by the latest design.

Reply:
Ack